pyfr/integrators/dual/pseudo/base.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+
+from collections import defaultdict
+
+from pyfr.integrators.base import BaseCommon
+from pyfr.util import proxylist
+
+
+class BaseDualPseudoIntegrator(BaseCommon):
```

Comment:
Is there any way we can move some more code out to BaseCommon to reduce duplication?